lib/conjur/resource.rb

```diff
@@ -274,36 +276,38 @@ def annotations
     end
     alias tags annotations
 
-    # @api private
-    # This is documented by Conjur::API#resources.
-    # Returns all resources (optionally qualified by kind) visible to the user with given credentials.
-    #
-    #
-    # Options are:
-    # - host - authz url,
-    # - credentials,
-    # - account,
-    # - owner (optional),
-    # - kind (optional),
-    # - search (optional),
-    # - limit (optional),
-    # - offset (optional).
-    def self.all options = {}
-      host, credentials, account, kind = options.values_at(*[:host, :credentials, :account, :kind])
-      fail ArgumentError, "host and account are required" unless [host, account].all?
-      %w(host credentials account kind).each do |name|
-        options.delete(name.to_sym)
-      end
+    class << self
```

Comment:
This move makes it hard to see what changed (did anything?). Is there any specific reason for changing this style?